### PR TITLE
adds test example for hello_update.py

### PR DESCRIPTION
**Pull request by yashvanzara**
## What was changed
adds test example for hello_update.py

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: tests/hello/hello_update_test.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            GreetingWorkflow.run, id=str(uuid.uuid4()), task_queue=task_queue_name
+        )
+
+        assert WorkflowExecutionStatus.RUNNING == (await handle.describe()).status
+
+        update_result = await handle.execute_update(
+            GreetingWorkflow.update_workflow_status
+        )
+        assert "Workflow status updated" == update_result
+        assert WorkflowExecutionStatus.COMPLETED == (await handle.describe()).status
```
Comment (cretz) [inline tests/hello/hello_update_test.py:22]: You'll need to `await handle.result()` before this line to ensure this test isn't flaky

**Review by cretz — COMMENTED** [review]
Thanks! Just a couple of minor things

**Comment by cretz** [comment]
This test looks good, thanks! But I think CI is failing for a known update bug fix we haven't released yet. In https://github.com/temporalio/sdk-python/pull/521 we made sure that workflows and updates completed in the same task are properly handled. You may be able to update to Python SDK 1.6.1 to check (update `pyproject.toml` to `1.6.1` then `poetry update temporalio`), but otherwise it may have to wait until next release.

File: tests/hello/hello_update_test.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import uuid
+
+from temporalio.client import Client, WorkflowExecutionStatus
+from temporalio.worker import Worker
+
+from hello.hello_update import GreetingWorkflow
+
+
+async def test_update_workflow(client: Client):
```
Comment (cretz) [inline tests/hello/hello_update_test.py:9]: Sorry it took so long to get back to this. There is an error in our time-skipping test server wrt updates. Can you skip it?

```suggestion
async def test_update_workflow(client: Client(client: Client, env: WorkflowEnvironment):
    if env.supports_time_skipping:
        pytest.skip("Time-skipping test server currently has issue with update: https://github.com/temporalio/sdk-java/issues/1903")
```

Comment (yashvanzara) [reply tests/hello/hello_update_test.py]: Done

**Review by cretz — COMMENTED** [review]
Make sure you reformat (e.g. `poe format`), CI is currently failing

**Comment by yashvanzara** [comment]
> Make sure you reformat (e.g. `poe format`), CI is currently failing

Yep, my bad - I had just accepted the suggestion you shared - fixed now

**Review by cretz — APPROVED** [review]
Looks great, thanks!

**Pull request merged**